src/main/java/uk/gov/hmcts/probate/service/DormantCaseService.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.probate.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.probate.model.ccd.CcdCaseType;
+import uk.gov.hmcts.probate.model.ccd.EventId;
+import uk.gov.hmcts.probate.model.ccd.raw.request.ReturnedCaseDetails;
+import uk.gov.hmcts.probate.security.SecurityUtils;
+import uk.gov.hmcts.probate.service.ccd.CcdClientApi;
+import uk.gov.hmcts.reform.probate.model.cases.grantofrepresentation.GrantOfRepresentationData;
+
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.List;
+
+import static uk.gov.hmcts.probate.model.Constants.MOVED_INTO_DORMANT_TIME_ELAPSED;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class DormantCaseService {
+
+    public static final String DORMANT_SUMMARY = "This case has been moved to "
+            + "the dormant state due to no action or event on the case for 6 months";
+    public static final String REACTIVATE_DORMANT_SUMMARY = "Case-reactivated due to new evidence received";
+    private final CaseQueryService caseQueryService;
+    private final CcdClientApi ccdClientApi;
+    private final SecurityUtils securityUtils;
+
+    public void makeCasesDormant(String date) {
+        log.info("Make Dormant for date: {}", date);
+        List<ReturnedCaseDetails> cases = caseQueryService.findCaseToBeMadeDormant(date);
+        log.info("Found {} cases with dated document for Make Dormant", cases.size());
+        for (ReturnedCaseDetails returnedCaseDetails : cases) {
+            GrantOfRepresentationData grantOfRepresentationData = GrantOfRepresentationData.builder()
+                    .movedIntoDormantDateTime(LocalDateTime.now(ZoneOffset.UTC)
+                            .plusMinutes(MOVED_INTO_DORMANT_TIME_ELAPSED))
+                    .build();
+            log.info("Updating case to Dormant in CCD by scheduler for case id : {}",returnedCaseDetails.getId());
+            ccdClientApi.updateCaseAsCaseworker(CcdCaseType.GRANT_OF_REPRESENTATION,
+                    returnedCaseDetails.getId().toString(),
+                    grantOfRepresentationData, EventId.MAKE_CASE_DORMANT,
+                    securityUtils.getUserByAuthTokenAndServiceSecurityDTO(), "", DORMANT_SUMMARY);
```

Comment:
why didnt we add the same summary as a description here - you have a "" instead

Reply:
Added in the description